modules/tensor_mechanics/src/kernels/StressDivergenceTensors.C

```diff
@@ -14,9 +14,7 @@ InputParameters validParams<StressDivergenceTensors>()
   InputParameters params = validParams<Kernel>();
   params.addClassDescription("Stress divergence kernel (used by the TensorMechanics action)");
   params.addRequiredParam<unsigned int>("component", "An integer corresponding to the direction the variable this kernel acts in. (0 for x, 1 for y, 2 for z)");
-  params.addCoupledVar("disp_x", "The x displacement");
-  params.addCoupledVar("disp_y", "The y displacement");
-  params.addCoupledVar("disp_z", "The z displacement");
```

Comment:
Here you are removing the ```disp_*``` parameters entirely rather than deprecating them linek mentioned above. This is fine by me, but it is going to break a lot of apps, just so you know.